src/components/Banner/AnonBannerSticky.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { DesktopOnly, MobileOnly } from './../Responsive'
+import BannerDesktop from './BannerDesktop'
+import BannerMobile from './BannerMobile'
+import StickyBannerContent from './StickyBannerContent'
+
+class AnonBannerSticky extends React.PureComponent {
+  state = { isVisible: false }
+
+  componentDidMount () {
+    this.viewportHeight = window.innerHeight
+  }
+
+  componentDidUpdate (prevProps) {
+    if (!prevProps.bannerStaticRef && this.props.bannerStaticRef) {
+      this.checkVisibility()
+      this.timer = setTimeout(this.checkVisibility, 50)
+    }
+  }
+
+  componentWillUnmount () {
+    clearTimeout(this.timer)
+  }
+
+  checkVisibility = () => {
+    const { bannerStaticRef } = this.props
+
+    if (bannerStaticRef) {
+      const bannerTop = bannerStaticRef.getBoundingClientRect().top + 100
+      const isVisible = bannerTop <= this.viewportHeight && bannerTop >= 0
```

Comment:
Static banner is always at the bottom of the page -> `bannerTop >= 0` is redundant